scripts/snapshot-framework-stats.py

```diff
@@ -26,6 +26,33 @@ def get_stats(data_client, framework_slug):
     return data_client.get_framework_stats(framework_slug)
 
 
+def log_human_readable_stats(stats):
+    total_applications = 0
+    made_declaration = 0
+    added_services = 0
+    completed = 0
+    started = 0
+
+    for stat in stats['interested_suppliers']:
+        total_applications += stat['count']
+        if stat['has_completed_services']:
+            if stat['declaration_status'] == 'complete':
+                completed += stat['count']
+            elif stat['declaration_status'] == 'started':
+                added_services += stat['count']
+        else:
+            if stat['declaration_status'] == 'complete':
+                made_declaration += stat['count']
+            else:
+                started += stat['count']
+
+    logger.info(f"Started: {started}")
```

Comment:
Could we have one or two blank lines before and after this printout so it's easy to see?

Reply:
I can go one better than that!

Reply:
😁 for a second I was bracing myself for some emojii 😆 

This is good too though!